src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/RoboticsDataMapper.java

```diff
@@ -97,12 +98,20 @@ private CaseHeader buildCaseHeader(CaseData caseData, String authToken) {
             .caseNumber(caseData.getLegacyCaseReference())
             .owningCourtCode("807")
             .owningCourtName("CCMCC")
-            .caseType("PERSONAL INJURY")
+            .caseType(getCaseType(caseData))
             .preferredCourtCode(locationRefDataUtil.getPreferredCourtData(caseData, authToken, true))
             .caseAllocatedTo(buildAllocatedTrack(caseData.getAllocatedTrack()))
             .build();
     }
 
+    private String getCaseType(CaseData caseData) {
+        if (caseData.getClaimTypeUnSpec() != null
+            && caseData.getClaimTypeUnSpec().name().equals(ClaimTypeUnspec.PERSONAL_INJURY.name())) {
```

Comment:
you could get away from the null check with only ClaimTypeUnspec.PERSONAL_INJURY.name().equals(caseData.getClaimTypeUnSpec()